include/sound/sof/header.h

```diff
@@ -130,7 +130,7 @@
 #define SOF_IPC_MESSAGE_ID(x)			((x) & 0xffff)
 
 /* maximum message size for mailbox Tx/Rx */
-#define SOF_IPC_MSG_MAX_SIZE			384
+#define SOF_IPC_MSG_MAX_SIZE			0x10000
```

Comment:
64K?.. Wow, that's a bold jump...

Reply:
@lyakh go big, or go home!

Reply:
it is just for size validation, not real size

Reply:
4k max please.

Reply:
sorry, I am confused by math.  On CNL, it is 0x1000, should be 4KB, and on TGL, it is 0x2000, should be 8KB.  Now update it to 16KB for future.